modules/backup/main.tf

```diff
@@ -3,12 +3,20 @@ locals {
 }
 
 resource "aws_backup_vault" "default" {
+
+  # KMS used to encrypt is at an account level so not included here.
```

Comment:
This is not correct. The KMS needs to be specified in order for the backup to use it for encryption. This module currently does not allow for it, so rather than suppressing it, this should be fixed.

Reply:
We looked at this last week. KMS keys are unique to each environment so we cannot actually pick them up in this code as we don't know which one it is using. Not sure how we can fix something that we can't find

Reply:
The KMS arn could be passed as a variable and in the MP repo we could retrieve the KMS keys dynamically, but this is probably an overkill.

Could you update the comment so that it is correct though and so it is all in one line, for example:
#checkov:skip=CKV_AWS_166: "Ensure Backup Vault is encrypted at rest using KMS CMK - Tricky to implement, hence using AWS managed KMS key."

Reply:
Change made